conans/client/graph/graph_builder.py

```diff
@@ -214,18 +214,19 @@ def _handle_require(self, name, node, require, dep_graph, check_updates, update,
                                 update, remotes, processed_profile, graph_lock)
 
     @staticmethod
-    def _conflicting_references(previous_ref, new_ref, consumer_ref=None):
-        if previous_ref.copy_clear_rev() != new_ref.copy_clear_rev():
+    def _conflicting_references(previous, new_ref, consumer_ref=None):
+        if previous.ref.copy_clear_rev() != new_ref.copy_clear_rev():
             if consumer_ref:
                 raise ConanException("Conflict in %s\n"
-                                     "    Requirement %s conflicts with already defined %s\n"
+                                     "    Requirement %s conflicts with already defined %s in %s\n"
                                      "    To change it, override it in your base requirements"
-                                     % (consumer_ref, new_ref, previous_ref))
+                                     % (consumer_ref, new_ref, previous.ref,
+                                        next(iter(previous.dependants)).src))
```

Comment:
Why only the first dependant?

Reply:
This still sounds wrong. The first dependant is not necessarily the one causing the conflict (it can have the version overriden). The second one is not the one being built, could be just another package in the graph.

Reply:
I see, but as described on https://github.com/conan-io/conan/issues/5617#issue-480105893, is hard to track when some conflict occurs. Showing both package and their linked node (package), helps to understand the problem. About the override feature, that's true, but when overriding the version we won't see this error.